### PR TITLE
Fix issue https://github.com/kumparan/react-native-android-inapp-updates/issues/13

### PR DESCRIPTION
**Pull request by HorbachAndrii**

### CODE REVIEW

File: package.json
Comment (gedeagas) [inline package.json]: Hi @HorbachAndrii thank you for your awesome pr, can you revert the versioning to default?. @kumparan/react-native will handle the deployment and versioning to NPM. cc @renopp

File: android/src/main/java/com/agastya/androidinappupdates/AndroidInappUpdatesModule.java
Comment (gedeagas) [inline android/src/main/java/com/agastya/androidinappupdates/AndroidInappUpdatesModule.java:66]: @renopp can you help to test and confirm the result?

File: package.json
Comment (HorbachAndrii) [reply package.json]: Roger that)

Comment (gedeagas) [reply package.json]: Thank you for the swift response @HorbachAndrii.

**Comment by gedeagas** [comment]
LGTM from me 🚀. cc @renopp and the @kumparan/frontend team.

**Review by gedeagas — APPROVED** [review]

File: android/src/main/java/com/agastya/androidinappupdates/AndroidInappUpdatesModule.java
Comment (HorbachAndrii) [reply android/src/main/java/com/agastya/androidinappupdates/AndroidInappUpdatesModule.java:66]: I tested my project using [Internal App Sharing](https://play.google.com/console/internal-app-sharing) and this fix works fine.

Comment (gedeagas) [reply android/src/main/java/com/agastya/androidinappupdates/AndroidInappUpdatesModule.java:66]: @HorbachAndrii hey yes i think this should work fine, but as per kumparan policy any external contribution need to be reviewed by internal employee before merging it. Since I am no longger an employee at kumparan i can't merge this.

Comment (HorbachAndrii) [reply android/src/main/java/com/agastya/androidinappupdates/AndroidInappUpdatesModule.java:66]: I hope it doesn't take long

**Comment by ferdianr** [comment]
thanks for the contributions @HorbachAndrii and @gedeagas. we will patch it soon.

**Pull request merged**
